butterfree/configs/db/kafka_config.py

```diff
@@ -0,0 +1,131 @@
+"""Holds configurations to read and write with Spark to Kafka."""
+from typing import Dict, List
+
+from butterfree.configs import environment
+from butterfree.configs.db import AbstractWriteConfig
+
+
+class KafkaConfig(AbstractWriteConfig):
+    """Configuration for Spark to connect to Kafka.
+
+    Attributes:
+        kafka_connection_string: kafka topic name.
```

Comment:
I think the correct description would be (by kafka_reader): 

```suggestion
        kafka_connection_string: string with hosts and ports to connect.
            The string need to be in the format: host1:port,host2:port,...,hostN:portN.
            The argument is not necessary if is passed as an environment variable
            named KAFKA_CONSUMER_CONNECTION_STRING.
```